auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -57,17 +60,86 @@
 
     private static final String TAG = WebAuthProvider.class.getName();
 
-    private static final String KEY_AUDIENCE = "audience";
-    private static final String KEY_SCOPE = "scope";
-    private static final String KEY_CONNECTION_SCOPE = "connection_scope";
-    private static final String SCOPE_TYPE_OPENID = "openid";
-    private static final String RESPONSE_TYPE_TOKEN = "token";
+    private static ResumableManager managerInstance;
 
-    private static OAuthManager managerInstance;
+    public static class LogoutBuilder {
 
+        private final Auth0 account;
+        private String scheme;
+        private CustomTabsOptions ctOptions;
+
+        LogoutBuilder(Auth0 account) {
+            this.account = account;
+
+            //Default values
+            this.scheme = "https";
+        }
+
+        /**
+         * When using a Custom Tabs compatible Browser, apply this customization options.
+         *
+         * @param options the Custom Tabs customization options
+         * @return the current builder instance
+         */
+        public LogoutBuilder withCustomTabsOptions(@NonNull CustomTabsOptions options) {
+            this.ctOptions = options;
+            return this;
+        }
+
+        /**
+         * Specify a custom Scheme to use on the Return To Uri. Default scheme is 'https'.
+         *
+         * @param scheme to use in the Callback Uri.
+         * @return the current builder instance
+         */
+        public LogoutBuilder withScheme(@NonNull String scheme) {
+            String lowerCase = scheme.toLowerCase();
+            if (!scheme.equals(lowerCase)) {
+                Log.w(TAG, "Please provide the scheme in lowercase and make sure it's the same configured in the intent filter. Android expects the scheme in lowercase");
```

Comment:
The same message is currently being used for the authentication builder. I've changed both to:
> Please provide the scheme in lowercase and make sure it's the same configured in the intent filter. Android expects the scheme to be lowercase.

This check is here to warn the developer if they run into callback issues that they have not used a scheme in lowercase. I'm not changing that for them because that would need to be changed not only on this method but also on the android manifest (which I cannot access from the code). So, we just print a friendly warning message for them to be aware they are doing it wrong.